readme.md

```diff
@@ -72,6 +72,22 @@ For Maven based projects add the following to your POM file in order to use MapS
 ...
 ```
 
+For Gradle, you need something along the following lines:
+
+```groovy
+plugins {
+    ...
+    id 'net.ltgt.apt' version '0.8'
+}
+dependencies {
+    ...
+    compile 'org.mapstruct:mapstruct:1.0.0.Final' // OR use this with Java 8 and beyond: org.mapstruct:mapstruct-jdk8:...
+
+    apt 'org.mapstruct:mapstruct-processor:1.0.0.Final'
```

Comment:
Is there any way we can get away with not specifying version.. Or stating somehow this is an example?